code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -378,7 +378,7 @@
 	addtimer(CALLBACK(src, PROC_REF(send_away)), 2 SECONDS)
 
 /obj/effect/temp_visual/teleporting_tornado/proc/send_away()
-	var/turf/ending_turfs = find_safe_turf()
+	var/turf/ending_turfs = find_safe_turf(zlevel = src.z)
```

Comment:
Don't know how I missed this one but I think the warp whistle shouldn't be limited to the current z, although maybe station turf would be fine? The description says "Calls a cloud to come pick you up and drop you at a random location on the station".

Reply:
will do